memestra/memestra.py

```diff
@@ -8,25 +8,27 @@
 
 
 # FIXME: this only handles module name not subpackages
-def resolve_module(module_name):
+def resolve_module(module_name, importer_path=None):
```

Comment:
if you default `importer_path` to '()' you can use `itertools.chain` here

Reply:
It's more a style / concision issue rather than a performance issue, yeah I think it's better.